tasks/actorstate/miner/sectorv7.go

```diff
@@ -0,0 +1,108 @@
+package miner
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel"
+	"go.uber.org/zap"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/lily/model"
+	minermodel "github.com/filecoin-project/lily/model/actors/miner"
+	"github.com/filecoin-project/lily/tasks/actorstate"
+)
+
+type V7SectorInfoExtractor struct{}
```

Comment:
`minermodel.MinerSectorInfoV7` gets its own extractor that we register with the appropriate actor code (v7 actors)